packages/ai/src/core.ts

```diff
@@ -76,35 +93,109 @@ export class EdgeDBAI {
     return response;
   }
 
-  async queryRag(
-    message: string,
-    context: QueryContext = this.context,
-  ): Promise<string> {
-    const response = await this.fetchRag({
-      model: this.options.model,
-      prompt: this.options.prompt,
+  async queryRag(query: string, context = this.context): Promise<string> {
+    const res = await this.fetchRag({
       context,
-      query: message,
+      query,
       stream: false,
     });
 
-    if (!response.headers.get("content-type")?.includes("application/json")) {
-      throw new Error(
-        "expected response to have content-type: application/json",
-      );
-    }
```

Comment:
Did we take this out because we just want to throw the existing exceptions that the `json` method returns?